auth-api/src/auth_api/models/task.py

```diff
@@ -64,6 +64,10 @@ def fetch_tasks(cls, task_search: TaskSearch):
             query = query.filter(Task.date_submitted <= task_search.end_date)
         if task_search.relationship_status:
             query = query.filter(Task.relationship_status == task_search.relationship_status)
+        if task_search.modified_by:
+            query = query.join(Task.modified_by) \
+                            .filter(text("lower(users.first_name || ' ' || users.last_name) like lower(:search_text)"))\
```

Comment:
I tried

```
query = query.join(Task.modified_by) \
                .filter(ilike_op(UserModel.firstname + text("' '") + UserModel.lastname, f'%{task_search.modified_by}'))
```

Didn't seem to return back the results even though when I printed the query.. it looked right..